spec/public-api.yaml

```diff
@@ -1,283 +1,243 @@
 openapi: 3.0.3
 info:
   title: Groundlight API
-  version: 0.6.0
-  description: Easy Computer Vision powered by Natural Language
+  version: 0.15.3
+  description: Groundlight makes it simple to understand images. You can easily create
+    computer vision detectors just by describing what you want to know using natural
+    language.
   contact:
-    name: Questions?
+    name: support@groundlight.ai
     email: support@groundlight.ai
 paths:
-  /me:
-    get:
-      operationId: Who am I
-      description: Retrieve the current user.
-      tags:
-        - user
-      security:
-        - ApiToken: []
-      responses:
-        "200":
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/User"
-          description: ""
-  /v1/actions/rules:
-    get:
-      operationId: List rules
-      description: Retrieve a list of rules.
-      parameters:
-        - name: page
-          required: false
-          in: query
-          description: A page number within the paginated result set.
-          schema:
-            type: integer
-        - name: page_size
-          required: false
-          in: query
-          description: Number of results to return per page.
-          schema:
-            type: integer
-      tags:
-        - rules
-      security:
-        - ApiToken: []
-      responses:
-        "200":
-          content:
-            application/json:
-              schema:
-                $ref: "#/components/schemas/PaginatedRuleList"
-          description: ""
-  /v1/actions/rules/{id}:
+  /v1/actions/detector/{detector_id}/rules:
     get:
-      operationId: Get rule
-      description: Retrieve a rule
+      operationId: List detector rules
+      description: List all rules for a detector
       parameters:
-        - in: path
-          name: id
-          schema:
-            type: integer
-          description: Get a rule by its ID.
-          required: true
+      - in: path
+        name: detector_id
+        schema:
+          type: string
+        required: true
       tags:
-        - rules
+      - actions
       security:
-        - ApiToken: []
+      - ApiToken: []
       responses:
-        "200":
+        '200':
           content:
             application/json:
               schema:
-                $ref: "#/components/schemas/Rule"
-          description: ""
-    delete:
-      operationId: Delete rule
-      description: Delete a rule
-      parameters:
-        - in: path
-          name: id
-          schema:
-            type: integer
-          description: Delete a rule by its ID.
-          required: true
-      tags:
-        - rules
-      security:
-        - ApiToken: []
-      responses:
-        "204":
-          description: ""
-  /v1/actions/detector/{detector_id}/rules:
+                $ref: '#/components/schemas/PaginatedRuleList'
+          description: ''
     post:
       operationId: Create rule
```

Comment:
```suggestion
      operationId: Create detector rule
```

Reply:
Just trying to add some parallelism to the name/description patterns

Reply:
I agreed with you at first, but then I realized this broke the parallelism to the delete rule endpoint. It's implicit that all rules belong to a detector